aws/resource_aws_ssm_association.go

```diff
@@ -39,6 +40,16 @@ func resourceAwsSsmAssociation() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"max_concurrency": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^([1-9][0-9]*|[1-9][0-9]%|[1-9]%|100%)$`), "must be a valid number (e.g. 10) or percentage including the percent sign (e.g. 10%)"),
+			},
+			"max_errors": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^([1-9][0-9]*|[1-9][0-9]%|[1-9]%|100%)$`), "must be a valid number (e.g. 10) or percentage including the percent sign (e.g. 10%)"),
```

Comment:
Looks like you might have copy pasted the regular expression from concurrency. The errors and concurrency regular expressions are different according to the [SSM API Reference](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_CreateAssociation.html#systemsmanager-CreateAssociation-request-MaxErrors) since errors allows a `0` or `0%` value:

```suggestion
				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^([1-9][0-9]*|[0]|[1-9][0-9]%|[0-9]%|100%)$`), "must be a valid number (e.g. 10) or percentage including the percent sign (e.g. 10%)"),
```